src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/CodeGen/ReadyToRunObjectWriter.cs

```diff
@@ -26,8 +28,8 @@ internal class ReadyToRunObjectWriter
         // Nodefactory for which ObjectWriter is instantiated for.
         private readonly NodeFactory _nodeFactory;
         private readonly string _objectFilePath;
+        private readonly EcmaModule _componentModule;
```

Comment:
Could you add a comment as to what `_componentModule` is for and when we should expect it to be null or not. It'll make it easier to reason about this member in the future.

Reply:
Good point, added XML comments for all the fields.